src/main/java/com/extrahardmode/features/monsters/CaveSpider.java

```diff
@@ -0,0 +1,75 @@
+
+package com.extrahardmode.features.monsters;
+
+
+import com.extrahardmode.ExtraHardMode;
+
+import com.extrahardmode.config.RootConfig;
+import com.extrahardmode.config.RootNode;
+import com.extrahardmode.module.EntityHelper;
+import com.extrahardmode.service.ListenerModule;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.World;
+import org.bukkit.block.Biome;
+import org.bukkit.block.BlockFace;
+import org.bukkit.entity.*;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.entity.CreatureSpawnEvent;
+
+/**
+ * Cave Spider
+ */
+public class CaveSpider extends ListenerModule
+{
+    private RootConfig CFG;
+
+
+    public CaveSpider(ExtraHardMode plugin)
+    {
+        super(plugin);
+    }
+
+
+    @Override
+    public void starting()
+    {
+        super.starting();
+        CFG = plugin.getModuleForClass(RootConfig.class);
+    }
+
+
+    /**
+     * When an Entity spawns: Spawn a Cave Spider sometimes instead of a spider in Swamps
+     *
+     * @param event which occurred
+     */
+    @EventHandler(priority = EventPriority.LOW)
```

Comment:
I pretty much copied this from the Witch method. Im sure it was a reason for being low.